### PR TITLE
create playground theme for RelatedPage

### LINKED ISSUE #231: Add `playground` theme for RelatedPage
- I noticed a few RelatedPage instances in /android where it linked to a playground - so we set it to the default.
- We could use this component in /api to highlight a playground for an API. Right now it's a small callout: 
image

cc @colleenmcginnis

### PR DESCRIPTION
**Pull request by katydecorah**
This PR adds a `playground` theme/contentType for RelatedPage. Example in use:

> image

## QA Checklist

- [x] No errors logged to console.
- [x] Component is accessible at mobile-size viewport.
- [x] Component is accessible at tablet-size viewport.
- [x] Component is accessible at laptop-size viewport.
- [x] Component is accessible at big-monitor-size viewport.

Open the test cases app locally on:

- [x] Chrome, no errors logged to console.
- [x] Firefox, no errors logged to console.
- [x] Safari, no errors logged to console.
- [x] Edge, no errors logged to console.
- [x] IE11, no errors logged to console.
- [x] Mobile Safari, no errors logged to console.
- [x] Android Chrome, no errors logged to console.

fixes #231

### CODE REVIEW

**Comment by katydecorah** [comment]
I went with the default theme only because the older interactive-tool.svg was already using those colors. I believe purple and yellow are left as color options - I do not have an opinion and am happy to update the icon/theme if someone does :)

File: src/components/related-page/related-page.js [hunk 1]
```diff
@@ -7,6 +7,45 @@ import BookImage from '../book-image/book-image';
 import GlossaryImage from '../glossary-image/glossary-image';
 import ExampleImage from '../example-image/example-image';
 
+class PlaygroundImage extends React.Component {
```
Comment (colleenmcginnis) [inline src/components/related-page/related-page.js]: Should we put `PlaygroundImage` into its own component and import to be consistent with `GlossaryImage`, `ExampleImage`, etc. above?

Comment (katydecorah) [reply src/components/related-page/related-page.js]: @colleenmcginnis my initial thought was that we likely wouldn't use the PlaygroundImage elsewhere, but that could change. I'll break it out into its own component!

**Comment by katydecorah** [comment]
@colleenmcginnis making PlaygroundImage was a good call, I found a lil bug with the svg. This is ready for review again!

**Pull request merged**